qmail-local.c

```diff
@@ -41,6 +42,20 @@ void temp_slowlock()
 void temp_qmail(fn) char *fn;
 { strerr_die5x(111,"Unable to open ",fn,": ",error_str(errno),". (#4.3.0)"); }
 
+/* writes ulong u in hex to char *s, does not NULL-terminate */
+unsigned int fmt_xlong(s,u) char *s; unsigned long u;
```

Comment:
C89 signatures, please.

Reply:
@xenotrope, as with #89, please let us know your preference to shepherd this through to approval and merge to master, or to back slowly away and have someone else handle that. Either way is totally fine.

Reply:
fmt_xlong() has been moved into its own file and compiled as part of the fs.a library along with the other fmt_* functions.

Reply:
fmt_xlong.c has has its len, q, and c variables broken into one-per-line to match the changes being made to genalloc.

Reply:
Conversion to C89 still missing.